arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -1459,32 +1378,21 @@
       "description": "A setting that contains all of the configuration for the automatic scaling of a resource."
     },
     "AutoscaleSettingResource": {
-      "properties": {
-        "id": {
-          "type": "string",
-          "description": "the resource identifier of the autoscale setting."
-        },
-        "name": {
-          "type": "string",
-          "description": "the name of the autoscale setting."
-        },
-        "location": {
-          "type": "string",
-          "description": "the location of the autoscale setting."
-        },
-        "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "the tags of the rule."
+      "allOf": [
+        {
+           "$ref": "#/definitions/Resource"
         },
-        "properties": {
-          "x-ms-client-flatten": true,
-          "$ref": "#/definitions/AutoscaleSetting",
-          "description": "the retrieved autoscale setting."
+        {
```

Comment:
please make sure that indentation is 2 spaces and tabs are replaced by spaces. Also take a look at the comment I made for RuleResource. A similar pattern should be followed over here.